### DISCUSSION TITLE
Saving shap.plots.text() output

### DISCUSSION THREAD

**Discussion by ketanmalempati** [shap/shap#3151] [General]
Hi,
I wanted to save the results of shap.plots.text() to my local and again use it with the interactions.
![image](https://github.com/shap/shap/assets/57043103/22f38d40-5197-4de6-a54a-6cd35ac528a3)
Any leads would help
Thank you

**Comment by ChatBotMatt** [comment]
You can save it as HTML (pass `display=False`) and then load that HTML in your browser. The interactiveness is preserved, though the resultant files can be very large.
